meal-voucher/build.gradle

```diff
@@ -44,5 +44,9 @@ dependencies {
     api project(':giftcard')
 
     // Tests
-    // TODO Add test implementations
+    testImplementation project(':test-core')
```

Comment:
We switched to `testImplementation testFixtures(project(':test-core'))` but it might be that the parent PR does not have these changes yet